### DISCUSSION TITLE
Order Faild

### DISCUSSION THREAD

**Discussion by c3aus3scu** [AshleyRedman/WPClothes2Order#94] [Q&A] [answered]
I don't know why after I make an Order in the Wocommerce Orders Tab I receive this:

#644 Marius-Vasile Adam | 4 hours ago | Order failed | Processing

**Comment by AshleyRedman** [comment]
Hi @c3aus3scu, I will need abit more information before I can help out with your issue. 
Can you provide some screenshot(s) of the WooCommerce order, so that I can see the order notes as the order notes will contain the response message from C2O. Can you also confirm is this is happening with only one product, or many?

**Comment by AshleyRedman** [comment, marked as answer]
Closing off due to inactivity, if you have any issues in the future, feel free to open an issue if you believe it to be a bug or new discussion for Q&A
